.golangci-sample.yml

```
@@ -0,0 +1,151 @@
+    - bin
+    - scripts
+  exclude-files:
+    - ".*\\.pb\\.go$"
+    - ".*_test\\.go$"
+  uniq-by-line: true
+
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+    check-blank: false # More relaxed for development
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)